docs/docsite/rst/getting_started_ee/run_execution_environment.rst

```diff
@@ -0,0 +1,89 @@
+.. _running_execution_environments:
+
+###############
+Running your EE
```

Comment:
Again, feels like this is suggesting the only way to run an EE is with navigator. Maybe it should be testing your EE?

Reply:
Fair point, sounds good to me

Reply:
On the other hand, people can think that navigator is for testing.
How about adding the [note](https://github.com/ansible/ansible-documentation/pull/18/commits/935dfa47b9a28bf473c41ee93f40769dea1d75b0) ?
If you think w/o the note and with changing it to `Testing` would be better, I'll change, np, let me know